src/bflib_sound.c

```diff
@@ -755,6 +760,7 @@ void play_atmos_sound(long smpl_idx)
     if (!S3DEmitterIsPlayingSample(Non3DEmitter, smpl_idx, 0))
     {
         S3DAddSampleToEmitterPri(Non3DEmitter, smpl_idx, 0, ATMOS_SOUND_PITCH, ATMOS_SOUND_VOLUME, 0, 3, 8, 0x7FFFFFFE);
+		SYNCDBG(9,"Playing atmos sound %d with pitch %d",(int)smpl_idx,(int)ATMOS_SOUND_PITCH);
```

Comment:
Indent seems off.

Reply:
It's not though, it's the same as the line above in the source file. Perhaps the github layout pushes it a tab to the right somehow.